pages/404html.js

```diff
@@ -0,0 +1,68 @@
+const error404 = `<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Blogium-Service / Not found route</title>
+	<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans:300,700">
+	<style>
+		body {
+			background: #0e141a;
+			font-family: 'Open Sans', sans-serif;
+		}
+
+		.container {
+			position: absolute;
+			top: 50%;
+			left: 50%;
+			transform: translate(-50%, -50%);
+			text-align: center;
+		}
+
+		h1 {
+			font-size: 72px;
```

Comment:
for font based dimensions, it's good to use `em` or `rem`.
Assuming the default font-size of a browser is `16px`, you can estimate `72px` as `4.5rem`
And this makes this layout responsive to user configurations to their browsers